examples/index.json

```
@@ -250,6 +250,10 @@
             {
                 "name": "Let Expression",
                 "url": "let-expression"
+            },
+            {
+                "name": "Configurable Variables",
+                "url": "configurables"
```

Shall we name this
```suggestion
                "url": "configurable"
```

instead?